concourse/pipelines/linux-image-build.jsonnet

```diff
@@ -262,6 +262,15 @@ local imgpublishjob = {
             load_var: 'publish-version',
             file: 'publish-version/version',
           },
+	  {
+	    task: 'generate-hash',
+	    file: 'guest-test-infra/concourse/tasks/generate-hash.yaml',
```

Comment:
we need to compute hash for windows images as well

Reply:
done